client/osnfs.go

```diff
@@ -0,0 +1,40 @@
+package client
+
+import (
+	"os"
+	"time"
+
+	"github.com/go-git/go-billy/v5"
+	osfs "github.com/go-git/go-billy/v5/osfs"
+)
+
+// NewOSFS returns a billy.FileSystem for a path.
+func NewOSFS(r string) billy.Filesystem {
+	bfs := osfs.New(r, osfs.WithBoundOS())
+	return bfs
+}
+
+// COS or OSFS + Change wraps a billy.FS to not fail the `Change` interface.
+type COS struct {
+	billy.Filesystem
+}
+
+// Chmod changes mode
+func (fs COS) Chmod(name string, mode os.FileMode) error {
+	return os.Chmod(fs.Join(fs.Root(), name), mode)
+}
+
+// Lchown changes ownership
```

Comment:
Extend for ignorant like me: … but does not dereference symbolic links

Reply:
done